hedera-node/hedera-app/src/test/resources/bootstrap.properties

```diff
@@ -185,7 +185,7 @@ hedera.recordStream.queueCapacity=5000
 hedera.recordStream.recordFileVersion=6
 hedera.recordStream.signatureFileVersion=6
 hedera.recordStream.logEveryTransaction=false
-hedera.recordStream.compressFilesOnCreation=false
+hedera.recordStream.compressFilesOnCreation=true
```

Comment:
Is `balances.compressOnCreation` intentionally set to `false` in this file? Or should it be updated to `true`?

Reply:
this is where I think I was expecting _your_ PR to land to change that.

Reply:
Correct. I reviewed this before mine was merged